Configuration/PyReleaseValidation/python/upgradeWorkflowComponents.py

```diff
@@ -7,9 +7,13 @@
     '2023D1',
     '2023D1PU',
     '2023D2',
-    '2023D2PU',
-    '2023D3',
+    '2023D2PU',    
+    '2023D3',    
     '2023D3PU',
+    '2023D2Timing',
+    '2023D2TimingPU',
+    '2023D3Timing',
+    '2023D3TimingPU'
```

Comment:
Nope, I'll add it.

On Mon, Aug 29, 2016 at 10:21 AM, Kevin Pedro <notifications@github.com>
wrote:

> In Configuration/PyReleaseValidation/python/upgradeWorkflowComponents.py
> <https://github.com/cms-sw/cmssw/pull/15613#discussion_r76626291>:
>
> >      '2023D3PU',
> > +    '2023D2Timing',
> > +    '2023D2TimingPU',
> > +    '2023D3Timing',
> > +    '2023D3TimingPU'
>
> Is there a reason to add timing versions for D3 and D2, but not D1?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/cms-sw/cmssw/pull/15613/files/ef3f4cf65ec984006646136a14c725d4e0245804#r76626291>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABBMOeITlRKgtp8qPD5tkjDLxtryshSCks5qkvj4gaJpZM4Jtm1g>
> .
>